CHANGES.rst

```diff
@@ -1,3 +1,18 @@
+Version 0.XX.X (20XX-XX-XX)
+===========================
+
+Instruments
+-----------
+- New TEXIO PSW-360L30 Power Supply (minimal implementation)
+
+Deprecated Features
+-------------------
```

Comment:
Now I know, why you did not find a "deprecated" section: It was added between your start of that pull request and the current state. Therefore the merge conflict.

Please merge the master branch and solve the conflict.

Reply:
I merged the master branch from the pymeasure repository and edited the CHANGES.rst file.